sections/semantics-sections.include

```diff
@@ -1648,6 +1645,10 @@
       preferences, the user agent implementor's preferences, etc.
     </div>
 
+<h6 id="code-fragments">Code fragments</h6>
+
+  <em>This section is non-normative.</em>
+
```

Comment:
The heading title is ambiguous - it depends on reading in context - so I'd like something different. 

Overall, I would prefer to change this to an example, but be able to refer to it with an id as you have done above.